app/jobs/email_suspended_members.rb

```diff
@@ -0,0 +1,19 @@
+class EmailSuspendedMembers
+  include Sidekiq::Worker
+  sidekiq_options queue: :members, retry: true, backtrace: true, failures: true 
+
+  def perform
+    suspended_members = Member.where(:status => "suspended")
```

Comment:
it should be noted that this only applied to symbols. objects cannot use this notation.